scripts/sync-helper.js

```
@@ -0,0 +1,221 @@
+
+function keysDiff(branch, release) {
+	const toMap = (ret, e) => {
+		ret[e] = true;
+		return ret;
+	};
+	const map1 = Object.keys(branch).reduce(toMap, {});
+	const map2 = Object.keys(release).reduce(toMap, {});
+	let changed = false;
+	for (let key in branch) {
+		if (!!!map2[key]) {
+			changed = true;
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)